operator/pkg/reconciliation/constructor.go

```diff
@@ -461,3 +291,172 @@ func calculatePodAntiAffinity(allowMultipleNodesPerWorker bool) *corev1.PodAntiA
+	return &corev1.Probe{
+		Handler: corev1.Handler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Port: intstr.FromInt(port),
+				Path: path,
+			},
+		},
+		InitialDelaySeconds: int32(initDelay),
+		PeriodSeconds:       int32(period),
+	}
+}
+
+func buildContainers(dc *api.CassandraDatacenter, serverVolumeMounts []corev1.VolumeMount) ([]corev1.Container, error) {
+	// cassandra container
+	cassContainer := corev1.Container{}
+	cassContainer.Name = "cassandra"
+
+	serverImage, err := dc.GetServerImage()
+	if err != nil {
+		return nil, err
+	}
+
+	cassContainer.Image = serverImage
+	cassContainer.Resources = dc.Spec.Resources
+	cassContainer.Env = []corev1.EnvVar{
+		{Name: "DS_LICENSE", Value: "accept"},
+		{Name: "DSE_AUTO_CONF_OFF", Value: "all"},
+		{Name: "USE_MGMT_API", Value: "true"},
+		{Name: "MGMT_API_EXPLICIT_START", Value: "true"},
+		// TODO remove this post 1.0
+		{Name: "DSE_MGMT_EXPLICIT_START", Value: "true"},
+	}
+
+	ports, err := dc.GetContainerPorts()
+	if err != nil {
+		return nil, err
+	}
+
+	cassContainer.Ports = ports
+	cassContainer.LivenessProbe = probe(8080, "/api/v0/probes/liveness", 15, 15)
+	cassContainer.ReadinessProbe = probe(8080, "/api/v0/probes/readiness", 20, 10)
+
+	cassServerLogsMount := corev1.VolumeMount{
+		Name:      "server-logs",
+		MountPath: "/var/log/cassandra",
+	}
+	serverVolumeMounts = append(serverVolumeMounts, cassServerLogsMount)
+	serverVolumeMounts = append(serverVolumeMounts, corev1.VolumeMount{
+		Name:      pvcName,
+		MountPath: "/var/lib/cassandra",
+	})
+	cassContainer.VolumeMounts = serverVolumeMounts
+
+	// server logger container
+	loggerContainer := corev1.Container{}
+	loggerContainer.Name = "server-system-logger"
+	loggerContainer.Image = "busybox"
+	loggerContainer.Args = []string{
+		"/bin/sh", "-c", "tail -n+1 -F /var/log/cassandra/system.log",
+	}
+	loggerContainer.VolumeMounts = []corev1.VolumeMount{cassServerLogsMount}
+
+	return []corev1.Container{cassContainer, loggerContainer}, nil
+}
+
+func buildInitContainers(dc *api.CassandraDatacenter) ([]corev1.Container, error) {
+	serverCfg := corev1.Container{}
+	serverCfg.Name = "server-config-init"
+	serverCfg.Image = dc.GetConfigBuilderImage()
+	serverCfgMount := corev1.VolumeMount{
+		Name:      "server-config",
+		MountPath: "/config",
+	}
+	serverCfg.VolumeMounts = []corev1.VolumeMount{serverCfgMount}
+
+	configData, err := dc.GetConfigAsJSON()
+	if err != nil {
+		return nil, err
+	}
+	serverVersion := dc.Spec.ServerVersion
+	serverCfg.Env = []corev1.EnvVar{
+		{Name: "CONFIG_FILE_DATA", Value: configData},
+		{Name: "POD_IP", ValueFrom: selectorFromFieldPath("status.podIP")},
+		{Name: "RACK_NAME", ValueFrom: selectorFromFieldPath(fmt.Sprintf("metadata.labels['%s']", api.RackLabel))},
+		{Name: "PRODUCT_VERSION", Value: serverVersion},
+		{Name: "PRODUCT_NAME", Value: dc.Spec.ServerType},
+		// TODO remove this post 1.0
+		{Name: "DSE_VERSION", Value: serverVersion},
```

Comment:
Do we still need this? The TODO says "remove this post 1.0" and we are post 1.0. Also, config-builder doesn't even reference it now: 

https://github.com/datastax/cass-config-builder/blob/master/src/com/datastax/configbuilder/operator.clj#L60-L68